src/main/java/de/tum/in/www1/artemis/domain/notification/ConversationNotification.java

```diff
@@ -0,0 +1,93 @@
+package de.tum.in.www1.artemis.domain.notification;
+
+import static de.tum.in.www1.artemis.domain.enumeration.NotificationType.*;
+
+import java.time.ZonedDateTime;
+import java.util.Set;
+
+import javax.persistence.*;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.enumeration.NotificationType;
+import de.tum.in.www1.artemis.domain.metis.Post;
+import de.tum.in.www1.artemis.domain.metis.conversation.Conversation;
+
+/**
+ * A Notification concerning all new messages in a conversation.
+ */
+@Entity
+@DiscriminatorValue("C")
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class ConversationNotification extends Notification {
+
+    @JsonIgnore
+    public static final Set<NotificationType> CONVERSATION_NOTIFICATION_TYPES = Set.of(CONVERSATION_NEW_MESSAGE);
+
+    @ManyToOne
+    @JoinColumn(name = "message_id")
+    private Post message;
+
+    @ManyToOne
+    @JoinColumn(name = "conversation_id")
+    private Conversation conversation;
+
+    @Transient
+    @JsonIgnore
+    public NotificationType notificationType;
```

Comment:
At the moment i do not even use, i removed it.  But normally this would allow to create notification without having to pass the type as an argument every time.